src/routes/docs/tutorials/angular/step-5/+page.markdoc

```diff
@@ -0,0 +1,51 @@
+---
+layout: tutorial
+title: Add navigation 
+description: Add navigation to your Angular applicating using Appwrite.
+step: 5
+---
+
+In our app we want to have a navigation bar that is always visible. We will add it to the `AppComponent` and use the `UserService` to show either:
+- A logout button if the user is logged in.
+- A login button if the user is not logged in.
+
+Update the `AppComponent` in `src/app/app.component.tsx`:
```

Comment:
I don't have this file. I have 3 files, are you missing setup instructions?
<img width="228" alt="Screenshot 2023-10-29 at 4 40 32 PM" src="https://github.com/appwrite/website/assets/29069505/d68c89ac-03d8-4f07-bcf9-b019911aaf48">

Reply:
Also should we really use TSX? What is the recommended default for Angular, can we stick to it?